src/Ryan_test_level.js

```diff
@@ -206,8 +210,13 @@ export class RyanLevel extends Phaser.Scene {
             this.player.getBody().y
         )
       
+        const sceneMusic = this.sound.add('sceneMusic');
+        if (!sceneMusic.isPlaying) {
+            sceneMusic.play();
+        }
+
       this.allSprites = this.children.list.filter(x => x instanceof Actor)
-      this.pauseHandler = handlePause(this, this.allSprites);
+      this.pauseHandler = handlePause(this, this.allSprites, sceneMusic);
```

Comment:
For future reference if you are passing this you don’t need to pass anything else you can use scene.sceneMusic and scene.allSprites I think!